src/Resolver/StoryResolver.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of Storyblok-Api.
+ *
+ * (c) SensioLabs Deutschland <info@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Storyblok\Api\Resolver;
+
+use Webmozart\Assert\Assert;
+
+/**
+ * @author Silas Joisten <silasjoisten@proton.me>
+ */
+final readonly class StoryResolver implements ResolverInterface
+{
+    public function resolve(array $target, array $relations): array
```

Comment:
should we ensure here, that those are real UUIDs ?

Reply:
Yea i think we can if i am not mistaken @edodusi wdyt?

Reply:
you mean here or validating the API response when we fetch a story object?

Reply:
There is an Assert which checks if the UUid key exists there we want to ensure that there is always a uuid coming back

Reply:
Done!